packages/ui-library/src/theme/components.ts

```diff
@@ -122,15 +122,25 @@ const baseComponents: Components = {
           borderLeftStyle: 'solid',
           borderTopLeftRadius: SPACING(4),
           borderBottomLeftRadius: SPACING(4),
+          paddingLeft: SPACING(8),
           borderColor: 'transparent',
         },
         ':last-child': {
           borderLeftStyle: 'solid',
           borderTopRightRadius: SPACING(4),
           borderBottomRightRadius: SPACING(4),
+          paddingRight: SPACING(8),
           borderColor: 'transparent',
         },
       },
+      head: {
+        ':first-child': {
+          paddingLeft: SPACING(8),
```

Comment:
This doesnt break any other table right? what about the portfolio one in the home?